test/validators.js

```diff
@@ -6435,6 +6435,10 @@ describe('Validators', () => {
         '+254728530234',
         '+299 12 34 56',
         '+94766660206',
+        '+41761112233',
```

Comment:
Could you make a new group of tests cases specifically for `de-CH`, other than adding here?

Reply:
Sure. Will fix this tomorrow!

Reply:
@profnandaa Done